main.py

```diff
@@ -0,0 +1,77 @@
+import os
+import json
+import requests
+import urllib.request
+from datetime import datetime
+
+from dotenv import load_dotenv
+from typing import Optional, Dict, List
+from fastapi import FastAPI, Response, Query
+
+import functions
+
+load_dotenv()
+app = FastAPI()
+
+
+@app.get("/comics/current")
+def getCurrentComic(response: Response):
+    try:
+        comicData = functions.loadJSON('https://xkcd.com/info.0.json')
+    except urllib.error.HTTPError as error:
+        response.status_code = error.code
+        return error
+    return functions.processComicData(comicData)
+
+
+@app.get("/comics/many")
+def getManyComics(response: Response, comic_ids: List[int] = Query(...)):
+    output = []
+    history = []
+    for comicId in comic_ids:
```

Comment:
there are some structures which take care of uniqueness for you - for example a dictionary and a set guarantee that all keys/elements are unique. Try using one of them here.